.gitattributes

```diff
@@ -0,0 +1,3 @@
+* text=auto
+*.png -text
+*.jpg -text
```

Comment:
The GitHub guide has the following for these lines:

```

# Denote all files that are truly binary and should not be modified.
*.png binary
*.jpg binary
```

Is `-text` equivalent? I think I find the GitHub way (together with the comment) more straight forward.

Reply:
I guess "not text" is the same as "binary". I just copied over from the source - the gitattributes documentation.

Reply:
Actually `binary` is equivalent to `-text -diff`. Since this patch is about line endings, I'd rather stick with the documentation.